serving/metadata-webhook/test/config/100-namespace.yaml

```diff
@@ -2,5 +2,3 @@ apiVersion: v1
 kind: Namespace
```

Comment:
Is this file useful anymore? I suspect its never going to be applied.

Reply:
Yes, it is as I deploy this on local to investigate CI issue.